### DISCUSSION TITLE
Budibase 3.0 launch week  🎉

### DISCUSSION THREAD

**Discussion by joebudi** [Budibase/budibase#14950] [Announcements]
For day one, we’ve just dropped a massive update, completely reimagining the way you build apps with our new data experience. 

Here’s what’s new:

✨ Enhanced Views – Control who sees what with powerful role-based access. 
🔧 Default Values – Automate data input and ensure consistency across your apps. 
⚡ Row Actions – Set user-specific actions for smarter workflows. 
🔗 View Joins – Connect data across tables effortlessly while keeping it secure. 
🎨 Dynamic Screen Generation – Instantly create screens from your data with a single click!

Ready to take your app-building to the next level? 

Dive into the full update to learn more.
https://budibase.com/blog/updates/v3/launchweek/day1/

Stay tuned for more updates this week! 👀

**Comment by joebudi** [comment]
Budibase 3.0 launch week continues! 🎉

Day 2 of the Budibase 3.0 launch week introduces Visual RBAC, a fresh and intuitive approach to managing user permissions.

Here’s what’s new:

🔒 Visual Role Management – Set up and view roles with an intuitive drag-and-drop editor.
🌿 Branched Permission Inheritance – Assign permissions through role hierarchies.
🛠️ Updated Role Definitions – Simplify permissions with new default role names and structures.

It's completely free to add custom roles to your Budibase apps.

Dive into the full update to learn more:
[https://budibase.com/blog/updates/v3/launchweek/day2](https://budibase.com/blog/updates/v3/launchweek/day2)

**Comment by joebudi** [comment]
Day three of the Budibase 3.0 Launch Week is here! Today, we’re happy to unveil a range of enhancements for Budibase’s Automation section. First up, we’ve added Branching, enabling you to define conditions for running sets of actions. Budibase Automations are now fully optimized for handling advanced end-to-end use-cases in a single flow.

Alongside this, we’ve added some key UX improvements for automating advanced workflows:
👉 Drag-and-Drop
🔍 Pan & Zoom
🧪 Updated Testing

There’s never been an easier way to create powerful, complex logic in Budibase. Check out our launch blog to find out more.

https://budibase.com/blog/updates/v3/launchweek/day3/

**Comment by joebudi** [comment]
It's day 4 of the Budibase Launch Week, and it's time to power your workflows with AI. Introducing 4 incredible AI features:

- Budibase AI
- AI configs
- AI CRON generator
- AI Column

Check them out:
https://budibase.com/blog/updates/v3/launchweek/day4/

**Comment by joebudi** [comment]
Day 5

What a week! I hope 3.0 has delivered on your expectations.

As we wrap up our first-ever launch week, we thought it would be the perfect moment to offer you an exciting glimpse into what's next for Budibase:

https://budibase.com/blog/updates/v3/launchweek/day5/

**Comment by andyburgessmd** [comment]
Thanks for all the nice new features. The new data features is already making a big difference to me, having the group by and calculations has been on my wish list for a while so great to have them.
